services/criterion/criterion.service.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+const { IMPROVED_STATUS, NOT_FOUND_STATUS } = require('./constants')
+
+const schema = Joi.string().required()
```

Comment:
Given the scope of possible statuses it's probably worth making the schema more explicit (unless you anticipate potential additional ones in the near future)
```suggestion
const schema = Joi.string()
  .allow(IMPROVED_STATUS, REGRESSED_STATUS)
  .required()
```